tree-sitter.d.ts

```diff
@@ -75,6 +75,10 @@ declare module "tree-sitter" {
   type ArrayCallback<TItem, TArray, TReturn> = (node: TItem, index?: number, array?: TArray) => TReturn
 
   export interface AstNodeArray extends ArrayLike<AstNode>, Iterable<AstNode> {
+    // Getters
+    isNamed: boolean;
```

Comment:
I think we shouldn't document this property; let's just treat it as an implementation detail of node arrays.

Reply:
I thought that too, I was just concerned about someone seeing it later down the line and thinking "Hey, this isn't documented, must be a bug". If only JS had true private methods =]